repository/src/test/java/org/alfresco/repo/domain/subscriptions/SubscriptionDAOTest.java

```diff
@@ -43,9 +43,9 @@
 import org.alfresco.service.transaction.TransactionService;
 import org.alfresco.test_category.OwnJVMTestsCategory;
 import org.alfresco.util.ApplicationContextHelper;
-import org.alfresco.util.testing.category.NeverRunsTests;
+import org.alfresco.util.testing.category.DBTests;
 
-@Category({OwnJVMTestsCategory.class, NeverRunsTests.class})
+@Category({OwnJVMTestsCategory.class, DBTests.class})
```

Comment:
Did you confirm that `testInsertAndDelete` test is run on your local machine or CI?